src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -0,0 +1,266 @@
+)
+public final class GenomicsDBImport extends GATKTool {
+
+  private static final long serialVersionUID = 1L;
+  private final long DEFAULT_SIZE_PER_COLUMN_PARTITION_PER_SAMPLE = 16*1024L;
+  private final long DEFAULT_SEGMENT_SIZE = 1048576L;
+  private final String DEFAULT_ARRAY_NAME_PREFIX = "genomicsdb_array";
+
+  @Argument(fullName = StandardArgumentDefinitions.GENOMICSDB_WORKSPACE_LONG_NAME,
+    shortName = StandardArgumentDefinitions.GENOMICSDB_WORKSPACE_SHORT_NAME,
+    doc = "Workspace where the database will be persisted. " +
+      "Must be either a local path or full path of a " +
+      "parallel file system like Lustre or NFS")
+  private String workspace = "";
+
+  @Argument(fullName = StandardArgumentDefinitions.GENOMICSDB_ARRAY_LONG_NAME,
+  shortName = StandardArgumentDefinitions.GENOMICSDB_ARRAY_SHORT_NAME,
+  doc = "TileDB array name used by GenomicsDB",
+  optional = true)
+  private String arrayName = DEFAULT_ARRAY_NAME_PREFIX;
+
+  @Argument(fullName = StandardArgumentDefinitions.GENOMICSDB_SEGMENT_SIZE_LONG_NAME,
+    shortName = StandardArgumentDefinitions.GENOMICSDB_SEGMENT_SIZE_SHORT_NAME,
+    doc = "Buffer size in bytes allocated for GenomicsDB attributes during " +
+      "import. Should be large enough to hold one variant data for one sample",
+    optional = true)
+  private
+  long segmentSize = DEFAULT_SEGMENT_SIZE;
+
+  @Argument(fullName = StandardArgumentDefinitions.VARIANT_LONG_NAME,
+  shortName = StandardArgumentDefinitions.VARIANT_SHORT_NAME,
+  doc = "GVCF files to be imported to GenomicsDB")
+  private List<String> variantFileNames;
+
+  @Argument(fullName = StandardArgumentDefinitions.SIZE_PER_COLUMN_PARTITION_PER_SAMPLE_LONG_NAME,
+  shortName = StandardArgumentDefinitions.SIZE_PER_COLUMN_PARTITION_PER_SAMPLE_SHORT_NAME,
+  doc = "Buffer size in bytes allocated to GenomicsDB variant readers. Default is 16KBytes." +
+    " Larger values are better as smaller values cause frequent disk writes",
+  optional = true)
+  private long sizePerColumnPartitionPerSample = DEFAULT_SIZE_PER_COLUMN_PARTITION_PER_SAMPLE;
+
+  @Argument(fullName = StandardArgumentDefinitions.GENOMICSDB_VID_MAP_FILE_LONG_NAME,
+  shortName = StandardArgumentDefinitions.GENOMICSDB_VID_MAP_FILE_SHORT_NAME,
+  doc = "JSON file including INFO/FORMAT/FILTER fields from headers and contig maps")
+  private String vidMapJSONFile = "";
+
+  @Argument(fullName = StandardArgumentDefinitions.GENOMICSDB_CALLSET_MAP_FILE_LONG_NAME,
+    shortName = StandardArgumentDefinitions.GENOMICSDB_CALLSET_MAP_FILE_SHORT_NAME,
+    doc = "JSON file including callset to GenomicsDB row maps")
+  private String callsetMapJSONFile = "";
+
+  @Argument(fullName = StandardArgumentDefinitions.GENOMICSDB_CREATE_WORKSPACE_LONG_NAME,
+  shortName = StandardArgumentDefinitions.GENOMICSDB_CREATE_WORKSPACE_SHORT_NAME,
+  doc = "Boolean flag - true means it will create GenomicsDB workspace, otherwise not",
+  optional = true)
+  private Boolean createWorkspace = false;
```

Comment:
Change this argument to "overwriteExistingWorkspace", defaulting to false. The tool should create a new workspace by default if the workspace doesn't already exist, but only overwrite an existing workspace if the user specifies this argument.